ntp-proto/src/algorithm/kalman/mod.rs

```diff
@@ -277,7 +277,7 @@ impl<C: NtpClock, SourceId: Hash + Eq + Copy + Debug> KalmanClockController<C, S
                 .slew_maximum_frequency_offset
                 .min(change.abs() / self.algo_config.slew_minimum_duration);
             let duration = Duration::from_secs_f64(change.abs() / freq);
-            info!(
+            debug!(
```

Comment:
The jump is still explicitly visible, made that choice because an actual jump is something the end user really might want to know about. Us fiddling with the frequency to reduce error less os.